ci/pipeline

```diff
@@ -284,38 +251,26 @@ def jenkins(): Unit = {
  * @return Version and artifact description of Marathon build.
  */
 @main
-def release(requestVersion: String, gitSha: String, latest: Boolean = false): Unit = {
-  val releaseVersion = withoutVersionPrefix(requestVersion)
+def release(args: String*): Unit = {
 
-  println(s"Releasing version: $releaseVersion")
-  %('git, "checkout", gitSha)
-
-  val version = build()
-  val tagVersion = version.toTagString
-  %('git, "tag", "-a", tagVersion, "-m", tagVersion)
+  val config = releases.parser.parse(args, releases.Config()).getOrElse {
+    sys.exit(1)
+    ???
```

Comment:
please add a `println(s"Couldn't parse....")`

Reply:
This should never be reached. Since Scopt will show this error

```
Error: Argument targets... failed when given 'foo'. foo is not a known release target; valid options are:
* docker-tag
* docker-latest
* s3-package
* linux-packages
* jar-artifact
* plugin-interface
Error: Argument targets... failed when given 'bar'. bar is not a known release target; valid options are:
* docker-tag
* docker-latest
* s3-package
* linux-packages
* jar-artifact
* plugin-interface
Error: Missing option --version
Error: Missing option --run-tests
pipeline release
Usage: scopt [options] [targets...]

  --version <value>    build that should be released, e.g. 1.7.42-deadbeef
  --run-tests <value>  whether to run tests for build
  targets...           targets to release: docker-tag, docker-latest, s3-package, linux-packages, jar-artifact, plugin-interface
Could not parse arguments 'foo, bar'.
```